cmd/upgrade/machinepool/cmd.go

```diff
@@ -53,7 +53,7 @@ var Cmd = &cobra.Command{
 	Args: func(_ *cobra.Command, argv []string) error {
 		if len(argv) != 1 {
 			return fmt.Errorf(
-				"expected exactly one command line parameter containing the id of the machine pool",
+				"machine pool id is missing",
```

Comment:
@Mischulee Is there a reason why the message is different and we are not verifying that the machine pool id matches the regex as we do in the other functions?

Reply:
In this case, for example this:.  ./rosa upgrade machinepool -c mparilov
The machinepool is not flag as in other cases, so we can not use the same function. But I can add additional check for regex.